src/travel-details-screens/components/TripSection.tsx

```diff
@@ -87,6 +97,7 @@ export const TripSection: React.FC<TripSectionProps> = ({
   const isWalkSection = leg.mode === Mode.Foot;
   const isBikeSection = leg.mode === Mode.Bicycle;
   const isFlexible = isLegFlexibleTransport(leg);
+  const timesAreApproximations = isFlexible;
```

Comment:
Might not be needed `timesAreApproximations` use isFlexible instead directly.

Reply:
Good to keep it like this I think. Both for readability and in case the condition doesn't remain exactly like this.